attendance/API_HR/c-hr.php

```diff
@@ -6166,18 +6166,40 @@ public static function getLeavesForYearMonth( $year, $month ){
         return $rows;
     }
 
+    public static function getWorkingDaysOfMonth( $year, $month ){
```

Comment:
why new function?????
existing function is already there, from which you can get working day,